version.go

```diff
@@ -1,4 +1,4 @@
 package iavl
 
 // Version of iavl.
-const Version = "0.11.1"
```

Comment:
this is a breaking change, so it should be `0.12.0` right?

Reply:
Is it really? What's breaking in the IAVL here? The GoAPI of the underlying db lib went through breaking change, but doesn't seem like there's a breaking change in this PR?

Reply:
Does IAVL only work with a given db implementation? My understanding was that it took in an interface, so others can in principle have supplied their own db implementation here.

Reply:
Great point